### PR TITLE
[#6330] remove testfairy

### LINKED ISSUE #6330: Remove TestFairy
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As user, I don't want my screen and seed phrase recorded so that I can maintain privacy and security.

### Description

For several weeks TestFairy has been [recording users seed phrase](https://github.com/status-im/status-react/issues/5567). TestFairy was also [recording seed phases back in April](https://github.com/status-im/status-react/issues/3876).

Status informs users that TestFairy is recording their screen, but it explicitly mentions that passwords are not recorded. We also don't mention this in the privacy policy.

We could fix the issue and hide the seed phrase, but if exposing sensitive info has already happened twice it will probably happen again. Better to fix the root cause.

Following our principles, and looking out for our users best interests we should remove TestFairy.  

#### Expected behavior
TestFairy removed from Status

#### Actual behavior
TestFairy in nightlies

### Solution

The solution is simple. Remove TestFairy.

### PR DESCRIPTION
**Pull request by rasom**
fix #6330 

status: ready

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

**Review by mandrigin — APPROVED** [review]
ltgm

**Review by dmitryn — APPROVED** [review]

**Review by chadyj — APPROVED** [review]
Thanks Roman for the quick response.

File: src/status_im/ui/screens/accounts/login/views.cljs [hunk 1]
```diff
@@ -68,7 +68,7 @@
          {:label             (i18n/label :t/password)
           :placeholder       (i18n/label :t/password)
           :ref               #(reset! password-text-input %)
-          :auto-focus        can-navigate-back? ;;this needed because keyboard overlays testfairy alert
+          :auto-focus        can-navigate-back?
```
Comment (flexsurfer) [inline src/status_im/ui/screens/accounts/login/views.cljs]: if there is no test-fairy anymore here should be always `true` and we don't need `can-navigate-back? [:can-navigate-back?]` anymore

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Comment by annadanchenko** [comment]
@rasom TF sessions are not recorded from this build, but I'm waiting for auto-test results and see there is some review comment to address, can you have a look at it, please?

File: src/status_im/ui/screens/accounts/login/views.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/screens/accounts/login/views.cljs]: @flexsurfer fixed

**Review by flexsurfer — APPROVED** [review]

**Review by yenda — APPROVED** [review]

**Comment by chadyj** [comment]
All good now @annadanchenko?

**Comment by arnetheduck** [comment]
that was quick!

**Pull request merged**